.github/ISSUE_TEMPLATE.md

```diff
@@ -1,12 +1,12 @@
-_For new Package Requests, see the guidelines_  
+_For new Package Requests, see the guidelines_
 
 ### Setup
-_Package Name:_ 
-_Package Version:_ 
+_Package Name:_
+_Package Version:_
 
-_NAS Model:_ 
-_NAS Architecture:_ 
-_DSM version:_ 
+_NAS Model:_
+_NAS Architecture:_
+_DSM version:_
```

Comment:
please do not remove trailing spaces in this file.
If issue form is filled without space after _ the italic font of the subjects does not work.